flake.nix

```diff
@@ -10,13 +10,26 @@
   outputs = { self, nixpkgs, nixlib }:
 
   # Library modules (depend on nixlib)
-  {
+  rec {
     # export all generator formats in ./formats
     nixosModules = nixlib.lib.mapAttrs' (file: _: {
       name = nixlib.lib.removeSuffix ".nix" file;
       # The exported module should include the internal format* options
       value.imports = [ (./formats + "/${file}") ./format-module.nix ];
     }) (builtins.readDir ./formats);
+
+    nixosGenerate = { system, pkgs, modules, format }:
```

Comment:
```suggestion
    # Example usage in flakes:
    # outputs = { self, nixpkgs, nixos-generators, ...}:  {
    #   vmware = nixos-generators.nixosGenerate { 
    #     pkgs = nixpkgs.legacyPackages.x86_64-linux;
    #     modules = [./configuration.nix];
    #     format = "vmware";
    #   };
    # }
    nixosGenerate = { pkgs, modules, format }:
```

Reply:
We don't need `system` as it is already implied by `pkgs`.

Reply:
Thanks looks good